client/signup/jetpack-connect/plans.jsx

```diff
@@ -156,16 +147,11 @@ const Plans = React.createClass( {
 							steps={ 3 } />
 
 						<div id="plans" className="plans has-sidebar">
```

Comment:
This gives me eslint warning. Maybe we could name the `div` above: `jetpack-connect__group` or `jetpack-connect__container` so we can call this `jetpack_connect__plans`. That should silence the warnings. :)